packages/databricks-vscode/src/ui/bundle-variables/VariableTreeNode.ts

```diff
@@ -0,0 +1,33 @@
+import {TreeItemCollapsibleState} from "vscode";
+import {BundleVariable} from "../../bundle/models/BundleVariableModel";
+import {BundleVariableTreeItem, BundleVariableTreeNode} from "./types";
+import {TreeItemTreeNode} from "../TreeItemTreeNode";
+
+export class VariableTreeNode implements BundleVariableTreeNode {
+    constructor(
+        public readonly key: string,
+        public readonly value: BundleVariable,
+        public readonly parent?: BundleVariableTreeNode
+    ) {}
+
+    getTreeItem(): BundleVariableTreeItem {
+        return {
+            label: this.key,
+            description:
+                this.value.vscodeOverrideValue ?? this.value.valueInTarget,
+            collapsibleState: TreeItemCollapsibleState.Collapsed,
+        };
+    }
+
+    getChildren(): BundleVariableTreeNode[] {
+        if (this.value.valueInTarget !== undefined) {
```

Comment:
It's actually possible to define global variables without descriptions, just with `var: value` as you do in targets. In that case we will get an expandable node, but it won't expand to anything

Reply:
The value is treaded as the default value in that case ig?

Reply:
Ouch, I commented under the wrong line. I mean that there are cases when we should use `TreeItemCollapsibleState.None`, as there's nothing to expand to

Reply:
But also if you define the global variable as simple `var: value`, we won't show the `value` in the tree item at all